### PR TITLE
feat(assets,events): add support for dataSetId.isNull filter

### PR DESCRIPTION
**Pull request by dmivankov**

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -1318,6 +1326,8 @@ export interface Node3DProperties {
   };
 }
 
+export type NullableFilter = { isNull: boolean };
```
Comment (polomani) [inline src/types/types.ts]: I'd suggest using already existing type.  it's on the line below:`NullableProperty<T>`

**Review by polomani — COMMENTED** [review]
try running `yarn lint --fix`, there are some issues

File: src/types/types.ts (hunk 1 above)
Comment (dmivankov) [reply src/types/types.ts]: NullableProperty is for two options T and isNull, for dataSetId.isNull there's no T option. Or is it legal to use NullablePropery<void> or something like that?

Comment (polomani) [reply src/types/types.ts]: ah, my bad. then it's fine,
just fix the linter issue

**Comment by BugGambit** [comment]
Shouldn't we include the other resources here? (sequences, time series, files etc.)

**Comment by dmivankov** [comment]
Other resources don't implement it yet, hopefully will add that soon. Probably will use the same sdk PR

**Comment by polomani** [comment]
@dmivankov also the tests are a bit exhaustive, i believe 1 test with either `isNull: true` or `false`  should be enough.

**Comment by polomani** [comment]
@dmivankov it's much easier than it seems
just run `yarn lint --fix` locally

**Comment by polomani** [comment]
@dmivankov looks good, are we waiting for something, or this can be released?

**Comment by dmivankov** [comment]
Waiting for python sdk review with similar change, then we can release both sdks + docs at roughly the same time. But if not separate releases also should work

**Review by polomani — APPROVED** [review]
wait, so we can merge with python at the same time

**Comment by andersfylling** [comment]
@dmivankov any updates on this?

**Pull request closed without merging**